docs/_documentations/remote-configuring.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: docs
+title: Configuring Codewind remote
+description: Configuring your environment to use Codewind remote
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing Cloud connection
+duration: 5 minutes
+permalink: remoteconfiguring
+type: document
+parent: installoncloud
+order: 1
+---
+
+# Configuring Codewind remote
+
+To use Codewind remote, configure your system by following these steps. 
+
+## Prerequisites
+- Install your preferred IDE on your local machine. For more information about installing Eclipse, see [Getting started with Codewind for Eclipse](mdteclipsegettingstarted.html), or for more information about installing VS Code, see [Getting started with Codewind for VS Code](mdt-vsc-getting-started.html).
+- Ensure that you have access to the Codewind CLI `cwctl`. To access the Codewind CLI, open a terminal window, and navigate to the following hidden folder: `~/.codewind/<version>` On Windows, navigate to the following folder: `/users/<username>/.codewind/<version>`.
```

Comment:
I think Windows file system path uses `\` instead of `/` and it's in C drive, so Windows path of `cwctl` would be `C:\Users\<user>\.codewind\<version>`, @lentheivendra can you confirm this since you tested it on Windows?

Reply:
Yup use use backslash for Windows.  The path in general would be `%SystemDrive%\\Users\\<username>\\.codewind\\<version>`

Reply:
@GeekArthur @lentheivendra Windows path updated on PR https://github.com/eclipse/codewind-docs/pull/257